tasks.py

```diff
@@ -54,7 +56,7 @@ def is_truthy(arg):
             "compose_files": [
                 "docker-compose.base.yml",
                 "docker-compose.redis.yml",
-                "docker-compose.dolt-hosted.yml",
+                # "docker-compose.dolt-hosted.yml",
```

Comment:
why this change?

Reply:
This should be left in there.

Reply:
Removed to have the default configuration local (non-hosted) only.